.github/workflows/publish_to_maven_central.yml

```diff
@@ -1,6 +1,13 @@
 name: Publish to Maven Central
 
 on:
+  workflow_dispatch:
+    inputs:
+      version-name:
```

Comment:
Out of curiosity, how would you pass this input to the job in a manual run?

Reply:
It looks similar to this, with one field.

![image](https://github.com/user-attachments/assets/22d0cf9f-51cd-43ed-ae64-a5a9d3ba7212)

Reply:
Nice!